projects/clr/hipamd/include/hip/amd_detail/amd_hip_bf16.h

```diff
@@ -111,9 +111,8 @@
 #if !defined(__HIPCC_RTC__)
 #include <hip/amd_detail/amd_hip_common.h>
 #include "amd_hip_vector_types.h"  // float2 etc
-#include "device_library_decls.h"  // ocml conversion functions
```

Comment:
We do have some `__ocml` api usage in header, I think currently this header is being included via some other header. But a good practice here is to include what you use. So that the other header (if) removes header we do not see a failure.